### PR TITLE
New Tutorial: Gradle for Kotlin/Native

### PR DESCRIPTION
**Pull request by hhariri**

### CODE REVIEW

File: pages/docs/tutorials/native/gradle-for-kotlin-native.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+Kotlin/Native has a plugin for [Gradle](https://gradle.org) which allows us to compile, build libraries, reference libraries, and in general perform all the actions that we would need to build our application. 
+
+In this tutorial we'll see how to
+
+* [Build an application with Gradle](#building-an-application)
+* [Build and reference a library with Gradle](#building-and-referencing-a-library)
+* [Generate CMake files to use with CLion](#generating-cmake-files)
+
+
+**Note**: This tutorial assumes somewhat familiarity with Gradle. If you're new to Gradle, while you should be able to follow along, it would be beneficial to review some of the
```
Comment (yole) [inline pages/docs/tutorials/native/gradle-for-kotlin-native.md]: "somewhat familiarity" is ungrammatical. "basic familiarity"?

File: pages/docs/tutorials/native/gradle-for-kotlin-native.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+apply plugin: 'konan'
+
+konan.targets = ['macbook']
+
+konanArtifacts {
+    program('hello')
+}
+```
+
+The `buildscript` section defines the repositories and dependencies required for the Gradle plugin for Kotlin/Native which is `konan`. Then comes the actual parameters for compiling:
```
Comment (yole) [inline pages/docs/tutorials/native/gradle-for-kotlin-native.md]: "then comes ... parameters" is ungrammatical; "then come" would be correct. Or rephrase.

File: pages/docs/tutorials/native/gradle-for-kotlin-native.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+![Gradle Run Result]({{ url_for('tutorial_img', filename='native/gradle-for-kotlin-native/gradle-run.png')}}))    
+
+## Generating CMake files
+
+In order to be able to use CLion, we need to have our project defined using CMake. The Gradle plugins provides us a convenient way to do this
+using the the command `generateCMake`:
+
+    gradle generateCMake
+    
+The result of running this action would give us a new file named `CMakeLists.txt` which allows us to open the project in CLion.
```
Comment (yole) [inline pages/docs/tutorials/native/gradle-for-kotlin-native.md]: "The result ... would give" is redundant. "The result .. is a new file" or "Running this action would give us a new file".

File: pages/docs/tutorials/native/gradle-for-kotlin-native.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+
+This is a simple taste of what the Gradle plugin for Kotlin/Native provides. For more detailed information about these, see the [corresponding README]https://github.com/JetBrains/kotlin-native/blob/fix-gradle-plugin-readme/GRADLE_PLUGIN.md.
```
Comment (yole) [inline pages/docs/tutorials/native/gradle-for-kotlin-native.md]: Why does this point to a branch?

File: pages/docs/tutorials/native/gradle-for-kotlin-native.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+
+    gradle build
+    
+    
+which will build the application. The end result should be the following:
+
+![Directory Structure After Build]({{ url_for('tutorial_img', filename='native/gradle-for-kotlin-native/dir-after-build.png')}}))
+     
+
+We can see that there is a new `build` directory with a subdirectory for each target. Inside this folder we have the actual executable with the name
```
Comment (yole) [inline pages/docs/tutorials/native/gradle-for-kotlin-native.md]: You use "folder" and "directory" interchangeably. Better to choose one term and stick to it.

File: pages/docs/tutorials/native/gradle-for-kotlin-native.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+In order to be able to use CLion, we need to have our project defined using CMake. The Gradle plugins provides us a convenient way to do this
+using the the command `generateCMake`:
+
+    gradle generateCMake
+    
+The result is a new file named `CMakeLists.txt` which allows us to open the project in CLion.
+
+
+This is a simple taste of what the Gradle plugin for Kotlin/Native provides. For more detailed information about these, see the [corresponding README]https://github.com/JetBrains/kotlin-native/blob/master/GRADLE_PLUGIN.md.
+
```
Comment (olonho) [inline pages/docs/tutorials/native/gradle-for-kotlin-native.md]: A lot of whitespaces at the end.

File: pages/docs/tutorials/native/gradle-for-kotlin-native.md [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+In order to be able to use CLion, we need to have our project defined using CMake. The Gradle plugins provides us a convenient way to do this
+using the the command `generateCMake`:
+
+    gradle generateCMake
+    
+The result is a new file named `CMakeLists.txt` which allows us to open the project in CLion.
+
+
+This is a simple taste of what the Gradle plugin for Kotlin/Native provides. For more detailed information about these, see the [corresponding README]https://github.com/JetBrains/kotlin-native/blob/master/GRADLE_PLUGIN.md.
```
Comment (ilmat192) [inline pages/docs/tutorials/native/gradle-for-kotlin-native.md]: Should the URL be in `(` and `)`?

**Pull request merged**